staging/2021/benchmarks/source.csv

```diff
@@ -511,14 +511,14 @@ Regaining Vision After Cataract Surgery,IRIS59,QCDR Measure,Outcome,Y,--,N,--,--
 "Failure to Progress (FTP): Proportion of patients failing to achieve a Minimal Clinically Important Difference (MCID) in improvement in pain score, measured via the Numeric Pain Rating Scale (NPRS), in rehabilitation patients with hip, leg or ankle (lower extremity except knee) injury.",IROMS14,QCDR Measure,Patient Reported Outcome,Y,--,N,--,--,--,--,--,--,--,--,N,N
 Failure to Progress (FTP): Proportion of patients failing to achieve a Minimal Clinically Important Difference (MCID) to indicate functional improvement in rehabilitation of patients with neck pain/injury measured via the validated Neck Disability Index (NDI).,IROMS15,QCDR Measure,Patient Reported Outcome,Y,--,N,--,--,--,--,--,--,--,--,N,N
 "Failure to Progress (FTP): Proportion of patients failing to achieve a Minimal Clinically Important Difference (MCID) in improvement in pain score, measured via the Numeric Pain Rating Scale (NPRS), in rehabilitation patients with neck pain/injury.",IROMS16,QCDR Measure,Patient Reported Outcome,Y,--,N,--,--,--,--,--,--,--,--,N,N
 Failure to Progress (FTP): Proportion of patients failing to achieve a Minimal Clinically Important Difference (MCID) to indicate functional improvement in rehabilitation patients with low back pain measured via the validated Modified Low Back Pain Disability Questionnaire (MDQ) score.,IROMS17,QCDR Measure,Patient Reported Outcome,Y,--,N,--,--,--,--,--,--,--,--,N,N
 "Failure to Progress (FTP): Proportion of patients failing to achieve a Minimal Clinically Important Difference (MCID) in improvement in pain score, measured via the Numeric Pain Rating Scale (NPRS), in rehabilitation patients with low back pain.",IROMS18,QCDR Measure,Patient Reported Outcome,Y,--,N,--,--,--,--,--,--,--,--,N,N
-"Failure to Progress (FTP): Proportion of patients failing to achieve a Minimal Clinically Important Difference (MCID) to indicate functional improvement in rehabilitation of patients with arm, shoulder, and hand injury measured via the validated Disability of Arm Shoulder and Hand (DASH) score, Quick Disability of Arm Shoulder and Hand (QDASH) score, or equivalent instrument which has undergone peer reviewed published validation and demonstrates a peer reviewed published MCID.",IROMS19,QCDR Measure,Patient Reported Outcome,--,N,--,--,--,--,--,--,--,--,N,N
+"Failure to Progress (FTP): Proportion of patients failing to achieve a Minimal Clinically Important Difference (MCID) to indicate functional improvement in rehabilitation of patients with arm, shoulder, and hand injury measured via the validated Disability of Arm Shoulder and Hand (DASH) score, Quick Disability of Arm Shoulder and Hand (QDASH) score, or equivalent instrument which has undergone peer reviewed published validation and demonstrates a peer reviewed published MCID.",IROMS19,QCDR Measure,Patient Reported Outcome,Y,--,N,--,--,--,--,--,--,--,--,N,N

```

Comment:
A fix for a previous data entry error in [QPPA-5949](https://github.com/CMSgov/qpp-measures-data/pull/512/files#diff-2e11ed9d9931a3e1e742c970871bddcc9bf78abbb30417f95a35e4ecd6a8f0b2L521)